_im-plugin/ism/policies.md

```diff
@@ -314,13 +314,13 @@ The `snapshot` operation has the following parameters:
 Parameter | Description | Type | Required | Default
 :--- | :--- |:--- |:--- |
 `repository` | The repository name that you register through the native snapshot API operations.  | `string` | Yes | -
-`snapshot` | The name of the snapshot. | `string` | Yes | -
+`snapshot` | The name of the snapshot. Accepts strings and the Mustache variables `{{ctx.index}}` and `{{ctx.indexUuid}}`. If the Mustache variables are invalid, then the snapshot name defaults to the index's name. | `string` or Mustache template | Yes | -
 
 ```json
 {
   "snapshot": {
     "repository": "my_backup",
-    "snapshot": "my_snapshot"
+    "snapshot": "{{ctx.IndexUuid}}"
```

Comment:
sorry my bad missed this one in past revision `ctx.IndexUuid` -> `ctx.indexUuid`